### ISSUE TITLE
Channels:write is not an option under Oauth section anymore

### ISSUE THREAD

**Issue by avrooyen** [jamietsao/slack-multi-channel-invite#12]
Thank you for this scripts.  Its going to make managing 100's of channels much easier.  One issue I ran into is I could only find these three options for Channnels in scopes (OAuth and Permissions).  I'm assuming they replaced the :write with :manage likely?

      - channels:read
      - channels:manage
      - channels:join

**Comment by jamietsao** [comment]
I believe you are trying to use a bot token vs a user token.  In the README, I make a mention of this:
```
Add the following permission scopes to a user token (bot tokens aren't allowed channels:write):
```

Here's the API I'm using that needs `channels.write`: https://api.slack.com/methods/conversations.invite

It mentions scopes that are needed for bot vs user tokens.  It looks like `channels:manage` will work for bot tokens.  To be honest, I don't know or remember the difference between the two.  Did you get the script to work?

**Comment by avrooyen** [comment]
Thank you for getting back.

I saw the note, assuming it was a deprecation and was not aware at the time of user vs bot tokens.  My mistake.   Of course, I did get the error cause it could not write channel.

I moved the tokens to user, added Channels:write and re-released the app.  It still seems to not be able to retrieve private (test1, test2) and public (test3) channels.

C:\Users\avroo\Documents\Slack Multi Add>slack-multi-channel-invite.exe -api_token=xoxb----------------------------------------------------------LFthfNrTF ***@***.******@***.***> -channels=test1,test2,test3 -private=true
Looking up users ...
Valid user (ID: U02D5H7LWLC) found for ***@***.***'
Inviting users to channels ...
Channel 'test1' not found -- skipping
Channel 'test2' not found -- skipping
Channel 'test3' not found -- skipping
All done! You're welcome =)

Debug shows its not getting Channels back.
Looking up users ...
Valid user (ID: U02D5H7LWLC) found for ***@***.***'
DEBUG: # of channels returned in page: 0
DEBUG: Total # of channels retrieved: 0

Channels in the right workspace:
***@***.***

OAuth Setup:
***@***.***

Any thoughts?

**Comment by jamietsao** [comment]
Hmm I'm not sure why no channels are being returned.  This API is being used to retrieve channels: https://api.slack.com/methods/conversations.list and it requires the `channels.read` permission, which I assume you've added to your user token?

**Comment by avrooyen** [comment]
Yes, I did add that.  I included a screenshot showing how all the scopes are setup.. maybe the screenshots were removed from the email reply.

I will try this a little later today.

Regards,

Abri van Rooyen | Chief Technology Officer (CTO)
Cobra Trading, Inc. | 3008 E. Hebron Pkwy | Building 400
Carrollton | TX | 75010 | 214-888-3044 | www.cobratrading.com<http://www.cobratrading.com/>

***@***.***
***@***.******@***.******@***.******@***.******@***.***<https://www.instagram.com/tradewithcobra/>

[encrypted]

**Comment by jamietsao** [comment]
@avrooyen - Any luck getting this to work?  Can I close this issue?

**Comment by avrooyen** [comment]
Up to you on closing.  I did not get around to test this due to moving to another urgent project att.  If you want to keep it open, I will go back and attempt this next week and give feedback.

**Comment by jamietsao** [comment]
@avrooyen - No problem, I'll leave it open.  Let me know if you still have issues.
